draft-oid4vc-haip-sd-jwt-vc.md

```diff
@@ -139,7 +139,42 @@ Note: Issuers should be mindful of how long the usage of the refresh token is al
 
 ### Wallet Attestation Schema {#wallet-attestation-schema}
 
-[Section 3.1 of wallet attestation draft would define the basics, and this profile will define the details.]
+Wallets MUST use attestations following the definition given in [!I-D.ietf-looker-key-attestation-client-authentication].
+
+In addition, the Wallet Attestation MUST contain the following claims:
```

Comment:
In addition to what? To the claims defined in draft-ietf-looker?

Reply:
In addition to "the definition given in [!I-D.ietf-looker-key-attestation-client-authentication]"